main.py

```diff
@@ -1,10 +1,57 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for player, info in players.items():
+        is_race_exist = Race.objects.filter(
+            name=info["race"]["name"]
+        ).exists()
+
+        is_guild_exist = Guild.objects.filter(
+            name=info["guild"]["name"]
+        ).exists() if info["guild"] else True
+
+        is_player_exist = Player.objects.filter(
+            nickname=player
+        ).exists()
+
+        if not is_race_exist:
+            Race.objects.create(
+                name=info["race"]["name"],
+                description=info["race"]["description"]
+            )
+        if not is_guild_exist:
+            Guild.objects.create(
+                name=info["guild"]["name"],
+                description=info["guild"]["description"]
+            )
+
+        players_race = Race.objects.get(name=info["race"]["name"])
+
+        for skill in info["race"]["skills"]:
+            if not Skill.objects.filter(name=skill["name"]):
+                Skill.objects.create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=players_race
+                )
+        if not is_player_exist:
+            Player.objects.create(
+                nickname=player,
+                email=info["email"],
+                bio=info["bio"],
+                race=players_race,
+                guild=Guild.objects.get(
```

Comment:
it not a good idea to get a guild in create(), it is hard to read. write it abowe